data/human/human missions.txt

```diff
@@ -6893,3 +6895,387 @@ mission "Lost Dog: Homeward Bound"
+				goto leave
+
+			label syndicate
+			`	"Well, I did say we had enemies in high places, didn't I? But don't worry. Bunker's a pro. We have an array of long-range stealth drones that do the spying. You won't be anywhere near them, though pirates do frequent the system, so you'll need to be quick."`
+
+			label leave
+			`	She waves to the server for the bill as she finishes the last gulp of her smoothie. "Let's go. Everyone will be meeting at the landing pad shortly."`
+			`	You go back to the <ship>'s landing pad, and sure enough, there are two people with backpacks and a few cases waiting for you. They introduce themselves as River and Bunker, and they enthusiastically stow their equipment away without a hitch.`
+			`	"Good. Now that that's done with, let's load up," says Bunker, before letting out a holler. "Taking down the Man. I love it!"`
+				accept
+
+			label out
+			`	"I'm sorry to hear that... I suppose it just wasn't meant to be." She gulps down her smoothie at lightning speed before leaving just as quickly.`
+				decline
+
+	on visit
+		dialog `You've landed on <planet>, but the filming equipment isn't in the system. Better depart and wait for your escorts to arrive in this star system.`
+
+
+
+mission "Adelita 3"
+	name "Film slave transaction"
+	description "Do a flyby of <destination> while Bunker spies on the slave transaction. You will be paid <payment> after you complete the entire job."
+	landing
+	cargo "stealth drones" 1
+	passengers 1
+	infiltrating
+	"apparent payment" 200000
+	source "Maker"
+	destination "Hippocrates"
+	to offer
+		has "Adelita 2: done"
+
+	on offer
+		conversation
+			`You drop off Ale and River at the spaceport, watching as they try to seem neither visibly embarrassed to be in their security uniforms nor uncomfortable with their disguises, even though they're now virtually indistinguishable from other Syndicate personnel. Bunker can't stop laughing at them. Everyone wishes each other luck as the two walk off, and you and Bunker go back to the <ship> and prepare for the journey to <planet>.`
+				accept
+
+	on visit
+		dialog `You realize that your escort carrying the drones hasn't entered the system yet. Better depart and wait for it.`
+
+
+
+mission "Adelita 4"
+	name "Take the film to <planet>"
+	description "Take the film to <destination>, where the rest of the Adelita group remained behind. You will be paid <payment> after you complete the entire job."
+	landing
+	passengers 1
+	cargo "stealth drones" 1
+	infiltrating
+	"apparent payment" 200000
+	source "Hippocrates"
+	destination "Maker"
+	to offer
+		has "Adelita 3: done"
+
+	on offer
+		conversation
+			branch "player has a fleet"
+				"total ships" - ( "ships: Fighter" + "ships: Drone" ) > 1
+
+			`It's not hard to find a remote canyon within range of the target area. Being so close to the surface of <origin> you're forced to deactivate your shields, but you keep the <ship> hovering in case you need to make a quick escape.`
```

Comment:
> Being so close to the surface of <origin> you're forced to deactivate your shields

What?

Reply:
If this is to justify getting your ship blasted by a few Quicksilvers, I don't think we should go this route. People already take issue with the Pirate Troubles mission where you can die under similar circumstances. Trying to add an explanation for why that can happen doesn't really work.

Either we should somehow take into account the strength of the player's ship to determine if you die outright or if you can escape with some scratches (new mechanic?), or we need to change the circumstances of the mission such that you could still die due to the Quicksilvers strafing you. Perhaps you need to move on foot to a location to deploy the drones, and you can get hit while outside your ship.

Reply:
> Being so close to the surface of you're forced to deactivate your shields

There are a few instances where _parked_ ships have deactivated shields; e.g. the R.N.S. Susquehanna on Hope & the guy you meet in Pug space who tells you a captain in an Argosy blew up a Pug ship with their shields down.

However, I get your point. I'm just saying ships deactivating their shields near the surfaces of planets is 100% not unprecedented.

Reply:
I'll replace it with you and Bunker going on foot.